agent.go

```diff
@@ -530,7 +532,7 @@ VERIFYCONSULSERVER:
 		if strings.Contains(err.Error(), "429") {
 			// 429 is a warning that something is unhealthy. This may occur when ESM
 			// races with Consul servers first starting up, so this is safe to retry.
-			a.logger.Printf("[ERR] Failed to query for Consul server versions (will retry): %v", err)
+			a.logger.Error("Failed to query for Consul server versions (will retry): ", "error", hclog.Fmt("%v", err))
```

Comment:
Can you remove the extra ": " here? Suggestion to change  `hclog.Fmt("%v", err)` => `err`